roles/openshift_excluder/defaults/main.yml

```diff
@@ -1,6 +1,15 @@
 ---
 # keep the 'current' package or update to 'latest' if available?
-openshift_excluder_package_state: present
-docker_excluder_package_state: present
+r_openshift_excluder_package_state: present
+r_openshift_excluder_docker_package_state: present
 
-enable_excluders: true
+# Inventory variables
+r_openshift_excluder_enable_excluders: "{{ openshift_enable_excluders | default(true) }}"
+r_openshift_excluder_enable_openshift_excluder: "{{ openshfit_enable_openshift_excluder | default(r_openshift_excluder_enable_excluders) }}"
+r_openshift_excluder_enable_docker_excluder: "{{ openshift_enable_docker_excluder | default(r_openshift_excluder_enable_excluders) }}"
```

Comment:
Need to preserve backwards compat with the old variables, can we add that here via additional defaulting?